components/brave_vpn/resources/panel/vpn_panel.tsx

```diff
@@ -18,6 +19,8 @@ import { PanelWrapper } from './style'
 import store from './state/store'
 import getPanelBrowserAPI from './api/panel_browser_api'
 
+setIconBasePath('chrome://resources/brave-icons')
```

Comment:
You don't seem to be using this at the moment - if you don't use the Leo `Icon` component, you don't need to do this.

You can also serve them from `chrome-untrusted://resources/brave-icons`

Reply:
Or `//resources/brave-icons` to match the scheme.

Reply:
Aha! I was curious why `chrome://resources/brave-icons/country-us.svg)` was set when I use `Icon`.
Changed to set `//resources/brave-icons` and `Icon` is used instead. Thanks!